bootstrap.d/dev-util.y4.yml

```diff
@@ -441,13 +441,15 @@ packages:
       subdir: 'ports'
       git: 'https://gitlab.com/sortix/os-test.git'
       branch: 'main'
-      commit: '645f26525bd1a83b6929c6ad253ea3c64aa7fad9'
+      commit: '83c313f7d6ae9d04533156ded05304f835fbf23e'
     tools_required:
       - system-gcc
     pkgs_required:
       - mlibc
       - make
-    revision: 1
+      - libiconv
+      - libintl
```

Comment:
I want to say that either the inclusion of make on this list is unneeded or you need to depend on gcc and binutils (if you want to compile on Managarm, but then I'd rather say that one needs to install base-devel). Or is it that we compile on the host and then do make run or similar on Managarm?

Reply:
You need make to invoke the test suite.